lib/routemaster/api_client.rb

```diff
@@ -25,16 +25,23 @@
 
 module Routemaster
   class APIClient
+
+    # Memoize the root resources at Class level so that we don't hit the cache
+    # all the time to fetch the root resource before doing anything else.
+    @@root_resources = {}
```

Comment:
Happy with the memoisation - but what's the plan to bust this cache when the interface changes? Are we leaving that problem to be solved lateR?